src/urllib3/contrib/emscripten/__init__.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+import urllib3.connection
+
+from ...connectionpool import HTTPConnectionPool, HTTPSConnectionPool
+from .connection import EmscriptenHTTPConnection, EmscriptenHTTPSConnection
+
+
+def _override_connections_for_emscripten() -> None:
```

Comment:
To match the terminology we previously used, let's name this `inject_into_urllib3()`.